src/main/scala/slamdata/SbtSlamData.scala

```diff
@@ -182,6 +187,29 @@ object SbtSlamData extends AutoPlugin {
 
   import autoImport._
 
+  override def globalSettings = Seq(
+    concurrentRestrictions in Global := {
```

Comment:
I think we can (and should) remove the `in Global` scoping since this is in `globalSettings`. Also nice catch.